run-full-test.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+
+date=`date +"%Y-%m-%d-%H.%M.%S"`
+builddir="build.$date"
+
+echo "####################################### running test suite"
+meson $builddir
+ninja -C $builddir test
+
+echo "####################################### running valgrind"
+pushd $builddir > /dev/null
+meson test --setup=valgrind --suite=valgrind
+popd > /dev/null
+
+echo "####################################### running ubsan"
+meson configure $builddir  -Db_sanitize=undefined
+ninja -C $builddir test
+
+echo "####################################### running asan"
+meson configure $builddir  -Db_sanitize=address
+ninja -C $builddir test
+
+echo "####################################### running scan-build"
+meson configure $builddir  -Db_sanitize=none
+ninja -C $builddir test
```

Comment:
I don't understand how this is running scan-build, but I could just be missing the obvious...

Reply:
whoops. that should read `ninja ... scan-build`. Fixed, will push in a second